examples/navier-stokes/index.rst

```diff
@@ -132,13 +132,80 @@ And its weak form is:
 
 .. math::
    :label: eq-weak-vector-ns
-   :nowrap:
 
    \begin{multline}
     \int_{\Omega} v \frac{\partial \boldsymbol{q}_N}{\partial t}  \,dV + \int_{\Gamma} v \widehat{\mathbf{n}} \cdot \boldsymbol{F} (\boldsymbol{q}_N) \,dS - \int_{\Omega} \nabla v\cdot\boldsymbol{F}(\boldsymbol{q}_N)\,dV  =
         \int_\Omega v \mathbf{S}(\boldsymbol{q}_N) \, dV \, , \; \forall v \in \mathcal{V}_p
    \end{multline}
 
+We solve equation :math:numref:`eq-weak-vector-ns` with Galerkin method discretization 
+which is the default option in our Navier-Stokes example.
+
+To resolve the numerical instability of the Galerkin method, we implement two stabilization 
+techniques (from :cite:`hughesetal2010`):
+
+- **SUPG** (streamline-upwind/Petrov-Galerkin)
+
+    In this method, the weighted residual of the original system of equations 
+    :math:numref:`eq-vector-ns` is added to the Galerkin formulation 
+    :math:numref:`eq-weak-vector-ns`. The weak form for this method is given as
+
+    .. math::
+       :label: eq-weak-vector-ns-supg
+
+       \begin{multline}
+          \int_{\Omega} v \, \frac{\partial \boldsymbol{q}_N}{\partial t}  \,dV + 
+          \int_{\Gamma} v \, \widehat{\mathbf{n}} \cdot \boldsymbol{F} \, (\boldsymbol{q}_N) \,dS - 
```

Comment:
Suggested change to make it consistent with the standalone libCEED example's notation (not only in this equation, but also in eq. (17) and the following one):

`\Gamma` -> `\partial \Omega`

Reply:
Thanks; done.